CHANGELOG.md

```diff
@@ -8,6 +8,18 @@ and this project adheres to
 
 ## Unreleased ([diff][unreleased-diff])
 
+## [0.3.2][] ([diff][0.3.2-diff]) - 2023-05-16
+
+### Added
+
+- Added support for custom query parameters
+
+### Deprecated
+
+- Deprecated ReadWithParameters
+- Deprecated ReadRawWithParameters
+- Deprecated DeleteWithParameters 
```

Comment:
we should remove before 1.0

Reply:
Definitely, I was thinking of removing in 0.4.0